Monotonless.Interfaces/IAutomator.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System;
```

Comment:
Great stuff

Reply:
Not putting the using statements inside of the namespace?? /s

Reply:
Hahaha thank you Danny for reminding me! I will get right on it ;)